includes/class-freemius.php

```diff
@@ -20992,23 +20992,43 @@ private function _sync_plugin_license(
                              */
                             $api = $this->get_api_site_scope();
 
-                            if ( ! self::$_global_admin_notices->has_sticky( 'api_blocked' ) ) {
-                                self::$_global_admin_notices->add(
-                                    sprintf(
-                                        $this->get_text_inline( 'Your server is blocking the access to Freemius\' API, which is crucial for %1$s synchronization. Please contact your host to whitelist %2$s', 'server-blocking-access' ),
-                                        $this->get_plugin_name(),
-                                        '<b>' . implode( ', ', $this->apply_filters( 'api_domains', array(
-                                            'api.freemius.com',
-                                            'wp.freemius.com'
-                                        ) ) ) . '</b>'
-                                    ) . '<br> ' . $this->get_text_inline( 'Error received from the server:', 'server-error-message' ) . var_export( $result->error, true ),
-                                    $this->get_text_x_inline( 'Oops', 'exclamation', 'oops' ) . '...',
-                                    'error',
-                                    $background,
-                                    'api_blocked'
-                                );
-                            }
+	                        $add_notice = false;
+	                        if ( $background ) {
+		                        $counter = (int) get_transient( '_fs_api_connection_retry_counter' );
+
+                                // We only want to show the notice after 3 consecutive failures.
+		                        if ( $counter >= 3 ) {

```

Comment:
@msladu please revert the order of the arguments, our convention is to put the constant on the left side.

Reply:
@vovafeldman Yes, my mistake. Changing the order.